Makefile

```diff
@@ -1,4 +1,4 @@
-altprodserver: migrate collectstatic ensurecrowdinclient downloadmessages compilemessages
+altprodserver: collectstatic ensurecrowdinclient downloadmessages compilemessages
```

Comment:
`collectstatic` runs `migrate`, so should we delete `migrate` in line 36 too?

Reply:
Oh no! Very good catch.